modules/visualization/dcv-eks/dcv-config/dcv-agent.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: dcv-agent
+  namespace: $NAMESPACE
+spec:
+  selector:
+    matchLabels:
+      env: dcv
+  template:
+    metadata:
+      labels:
+        env: dcv
+    spec:
+      restartPolicy: Always
+      serviceAccountName: dcv-agent-src-account
+      containers:
+        - name: nice-dcv-container
+          image: $IMAGE
+          command: [ "/usr/local/bin/run_script.sh" ]
+          imagePullPolicy: "Always"
+          env:
+            - name: ADDF_CONFIG_MAP_NAME
```

Comment:
Can you remove the ADDF prefix? Is it being referenced elsewhere? Try to use deployment_name and module_name wherever possible

Reply:
This is more of internal usage. I would suggest to use a relatively shorter name for better readability. I would use prefix of `DCV_EKS` for some level of visibility here.